person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    if is_of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  def of_age?
```

Comment:
- [ ] To run your code correctly the called `is_of_age?` must be equal to the original one. You have created a method `of_age?` but didn't call it correctly in `can_use_services?`. Please fix this issue.

`is_of_age?` = correct
`of_age?` = incorrect